### PR TITLE
fix typo in footnote

### PR DESCRIPTION
**Pull request by kapnobatai137**

### CODE REVIEW

**Comment by enki-hq** [comment]
Rule insight:author failed for files: 
  - linux/jobs-and-processes-data-manipulation/utilities/image-manipulation-using-convert.md

File: linux/jobs-and-processes-data-manipulation/utilities/image-manipulation-using-convert.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ $ ??? enki.png ???
 ---
 ## Footnotes
 [1:Where to get it]
-Part of ` imagemagick` suit of tools.
+Part of ` imagemagick` suite of tools.
```
Comment (kapnobatai137) [inline linux/jobs-and-processes-data-manipulation/utilities/image-manipulation-using-convert.md:85]:
```suggestion
Part of `imagemagick` suite of tools.
```

**Comment by enki-hq** [comment]
Rule insight:author failed for files: 
  - linux/jobs-and-processes-data-manipulation/utilities/image-manipulation-using-convert.md

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/linux/jobs-and-processes-data-manipulation/utilities/image-manipulation-using-convert.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/linux/jobs-and-processes-data-manipulation/utilities/image-manipulation-using-convert.md?ref=f34d69e11fb5c40480630edeaad89b8528b71ac7)

**Pull request merged**